### PR TITLE
Tickets/dm 8636

### PR DESCRIPTION
**Pull request by mssgill**
Changes just to the Task template, and not the others, for now.

### CODE REVIEW

File: templates/template.forTasks.rst [hunk 1]
```diff
@@ -5,96 +5,111 @@ Based on: https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type
...
-- Module membership
 
-Here we state what module implemented the task.
+     
+- Module membership:
+
+  Here we simply state in which module the task is implemented.
 
 See also
 =========
```
Comment (jonathansick) [inline templates/template.forTasks.rst]: You can leave out the header "See also" or else devs might get confused about whether they should have a header here in addition to the `.. seealso::` directive.

File: templates/template.forTasks.rst [hunk 2]
```diff
@@ -5,96 +5,111 @@ Based on: https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type
 Template for Tasks
 ##################
```
Comment (jonathansick) [inline templates/template.forTasks.rst]: Might want to style this after the title we intend. Like `ExampleTask`.

File: templates/index.rst [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+LSST DM Stack documentation
+============================
+
+Contents:
+
+.. toctree::
+   :maxdepth: 1
+
+   template.forTasks
```
Comment (jonathansick) [inline templates/index.rst]: If we make the title of the task template realistic, you can give it a sensible title here as:

```
Task Template <template.forTasks>
```

File: templates/template.forTasks.rst (hunk 1 above)
Comment (mssgill) [reply templates/template.forTasks.rst]: Ok, cool, done

File: templates/template.forTasks.rst (hunk 2 above)
Comment (mssgill) [reply templates/template.forTasks.rst]: Ok, done

File: templates/index.rst (hunk 3 above)
Comment (mssgill) [reply templates/index.rst]: Cool, did this, also

**Comment by jonathansick** [comment]
Looks like this work has moved to DM-8723.

**Pull request closed without merging**